Bang/GameEvents/CardEffects/States/WaitingMissedCardsAfterGatlingState.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Bang.Characters;
+using Bang.Players;
+using Bang.PlayingCards;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bang.GameEvents.CardEffects.States
+{
+    public class WaitingMissedCardsAfterGatlingState : HandlerState
+    {
+        private readonly Dictionary<Player, HandlerState> victimStates;
+        private readonly Game.Gameplay gameplay;
+
+        private Player currentVictim;
+
+        public WaitingMissedCardsAfterGatlingState(Dictionary<Player, HandlerState> victimStates, Game.Gameplay gameplay)
+        {
+            this.victimStates = victimStates;
+            this.gameplay = gameplay;
+        }
+        public override HandlerState ApplyCardEffect(Player player, BangGameCard card, Game.Gameplay gameplay)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override HandlerState ApplyReplyAction(BangGameCard card)
```

Comment:
This code duplicated the `WaitingMissedCardAfterBangState.ApplyReplyAction`  method. Use it directly.

Reply:
done